frontend/components/DistributionProgress.js

```diff
@@ -99,39 +107,21 @@ export default class DistributionProgress extends React.Component {
         color: distributedColor,
       });
       dataOuter.push({
-        name: "Bitcoin program",
-        value: parseInt(bitcoinProgram),
+        name: "Ecosystem Support",
+        value: parseInt(ecosystemSupport),
         color: distributedColor,
       });
       dataOuter.push({
-        name: "Stellar Build Challenge",
-        value: parseInt(buildChallenge),
+        name: "Use-Case Investment",
+        value: parseInt(useCaseInvestment),
         color: distributedColor,
       });
       dataOuter.push({
-        name: "Partnership program",
-        value: parseInt(partnershipProgram),
-        color: distributedColor,
-      });
-
-      let other = new BigNumber(availableCoins).minus(distributionAll);
-      dataOuter.push({
-        name: "Other",
-        value: other.toNumber(),
+        name: "User Acquisition",
+        value: parseInt(userAcquisition),
         color: distributedColor,
       });
 
-      dataOuter.push({
-        name: "Direct signup program left",
-        value: parseInt(directSignupLeft),
-        color: toDistributeColor,
-      });
-      dataOuter.push({
-        name: "Partnership program left",
-        value: parseInt(partnershipProgramLeft),
-        color: toDistributeColor,
-      });
-
```

Comment:
moving the 3 programs I was working with into the pie chart. do we need to update the table underneath?